examples/webgl_helpers.html

```diff
@@ -67,18 +69,18 @@
 					// To make sure that the matrixWorld is up to date for the boxhelpers
 					group.updateMatrixWorld(true);
 
-					var mesh = new THREE.Mesh( geometry, material );
 					group.add( mesh );
 
-					fnh = new THREE.FaceNormalsHelper( mesh, 5 );
-					scene.add( fnh );
+					// TODO: Implement BufferGeometry support in FaceNormalsHelper.
```

Comment:
I'm not sure it makes sense if `FaceNormalsHelper` supports `BufferGeometry`. The helper would need to calculate the face normals before visualization. I think it's better if helpers just visualize data which are actually present in the underlying object.

Reply:
Right. `FaceNormalsHelper` does not apply to `BufferGeometry`.

Reply:
OK, so just remove this bit from the example entirely?

Reply:
Removed 👍